### DISCUSSION TITLE
使用openai_api_demo/api_server.py结合其他的web界面时，如何使模型输出图片等其他格式的数据？

### DISCUSSION THREAD

**Discussion by skylark2262** [zai-org/ChatGLM3#1180] [Q&A] [answered]
在composite_demo/conversation.py中定义了image格式的输出
```
class Conversation:
    role: Role
    content: str
    tool: str | None = None
    image: Image | None = None
```
但是如果使用openai_api_demo/api_server.py结合非官方的web界面，比如[chatgpt-web](https://github.com/Chanzhaoyu/chatgpt-web/tree/main)该如何使模型回答非文本格式的数据呢？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
但是这个模型没有支持image啊支持image是用了代码解释器才行，api没有的
